### PR TITLE
[exporter] Fix issue in gpu exporter deployment

### PR DESCRIPTION
**Pull request by YanjieGao**
Fix issue in gpu exporter deployment

### CODE REVIEW

File: service-deployment/bootstrap/prometheus/node-exporter-ds.yaml.template [hunk 1]
```diff
@@ -41,12 +41,14 @@ spec:
...
       - image: {{ clusterinfo['dockerregistryinfo']['prefix'] }}gpu-exporter
+        securityContext:
+          privileged: true
         volumeMounts:
         - mountPath: /dev
           name: device-mount
         - mountPath: /var/drivers
           name: driver-path
-        - mountPath: /tmp
+        - mountPath: {{ clusterinfo['prometheusinfo']['node_exporter_textfile_dir'] }}
```
Comment (ydye) [inline service-deployment/bootstrap/prometheus/node-exporter-ds.yaml.template]: Pls use  clusterinfo[ 'dataPath' ]/export. Too much path parameter will make user confused.

File: service-deployment/src/gpu-exporter/start.sh.template [hunk 2]
```diff
@@ -17,6 +17,6 @@
 # DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-nohup /usr/local/./gpu_exporter -gpu_log_dir={{ clusterinfo['prometheusinfo']['node_exporter_textfile_dir'] }} > /usr/local/gpu_exporter.log &
+/usr/local/./gpu_exporter -gpu_log_dir={{ clusterconfig['clusterinfo']['prometheusinfo']['node_exporter_textfile_dir'] }} > /usr/local/gpu_exporter.log
```
Comment (ydye) [inline service-deployment/src/gpu-exporter/start.sh.template]: remove “/.” . And use clusterinfo[ 'dataPath' ]/export

File: prometheus/exporter/gpu_exporter.go [hunk 3]
```diff
@@ -152,7 +152,7 @@ func writeMetricsToFile(index int, logDir string) {
 }
 
 func main() {
-    logDir := flag.String("gpu_log_dir", "/var/local/prom-exporters", "log written dir")   
+    logDir := flag.String("gpu_log_dir", "/datastorage/prom-exporters", "log written dir")
```
Comment (ydye) [inline prometheus/exporter/gpu_exporter.go]: Should this log path on the host to persistent log data? {{ clusterinfo[ 'dataPath' ] }}/promethues

Comment (YanjieGao) [reply prometheus/exporter/gpu_exporter.go]: This will override by the launch shell

**Pull request merged**